docs/advanced-features/workflows.md

```diff
@@ -354,6 +354,78 @@ With this Schema lists such as
 `["Fixed 1", "Any string", 123]` 
 are valid but `["Any string", 123]` are invalid.
 
+### Objects
+Quilt also supports the [`object` data type](https://json-schema.org/understanding-json-schema/reference/object.html).
+You can use `object` if you need to define a list of metadata values
+that are objects themselves by using the `properties` keyword on an object
+(key-value pairs).
+
+```json
+{
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "id": {
+              "default": 123,
+              "type": "number"
+            },
+            "name": {
+              "default": "Optional default value",
+              "type": "string"
+            }
+        }
+    }
+}
+```
+
+### Compound enums
```

Comment:
```suggestion
### Enums of arrays and objects
```

Reply:
Added explicit h4 titles for each example code snippets